src/linker/Linker.Steps/LinkAttributesParser.cs

```diff
@@ -235,7 +236,7 @@ CustomAttributeArgument[] ReadCustomAttributeArguments (XPathNavigator nav, Type
 			case MetadataType.Object:
 				var argumentIterator = nav.SelectChildren ("argument", string.Empty);
 				if (argumentIterator?.MoveNext () != true) {
-					_context.LogError ($"Custom attribute argument for 'System.Object' requires nested 'argument' node.", 1043);
+					_context.LogError (null, DiagnosticId.CustomAttributeArgumentForTypeRequiresNestedNode, "System.Object", "argument");
```

Comment:
Is there any usage where `null` is not used as the first argument?

Reply:
Yes - there are some - ideally all should be non-null, but that's probably for another change.

Reply:
For example the one right below.